infra/feast-operator/internal/controller/featurestore_controller_loglevel_test.go

```diff
@@ -246,3 +242,7 @@ var _ = Describe("FeatureStore Controller - Feast service LogLevel", func() {
 
 	})
 })
+
+func strPtr(str string) *string {
```

Comment:
Are you sure a one line function like this is needed?

Reply:
unfortunately yes... the alternative would be separate vars for each.